<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditWidget</class>
 <widget class="QWidget" name="CreditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="labelSum">
     <property name="text">
      <string>Sum</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="labelEndValue">
     <property name="text">
      <string>21.10.2100</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="labelStartValue">
     <property name="text">
      <string>21.10.2020</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelStart">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelEnd">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>End</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelName">
     <property name="text">
      <string>Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="editName"/>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="editSum"/>
   </item>
   <item row="2" column="3">
    <widget class="QComboBox" name="comboPeriod"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelPeriod">
     <property name="text">
      <string>Period</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelInterest">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Interest</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLabel" name="labelInterestValue">
     <property name="text">
      <string>Interest</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
